Scripts/gen_Primary.sh

```diff
@@ -148,18 +148,22 @@ elif [[ "${TYPE}" == "Piplus" ]]; then
   resampler=TargetPiStopResampler
 elif [[ "${TYPE}" == "Piminus" ]]; then
   resampler=TargetPiStopResampler
-  dataset=sim.${OWNER}.PiMinusFilter.${STOPS_CAMPAIGN}.art # since we prefilter these for a given time
+  dataset=sim.${OWNER}.${TYPE}StopsFilt.${STOPS_CAMPAIGN}.art # since we prefilter these for a given time
 else
   resampler=${TYPE}StopResampler
 fi
 
 if [[ "${TYPE}" == "Piminus" ]]; then
-  samweb list-files "dh.dataset=sim.mu2e.PiMinusFilter.MDC2020aj.art and event_count > 0"  > Stops.txt
+  samweb list-files "dh.dataset=$dataset and event_count > 0"  > Stops.txt
 else
   samweb list-definition-files $dataset  > Stops.txt
 fi
 # calucate the max skip from the dataset
-nfiles=`samCountFiles.sh $dataset`
+nfiles= 0
+if [[ "${TYPE}" == "Piminus" ]]; then
+  nfiles=`wc -l  Stops.txt`
+else
+  nfiles=`samCountFiles.sh $dataset`
```

Comment:
You don't need lines 162-164.
L166 will work by itself with PR 364 that was just merged.